### PR TITLE
Resolve error message in code editor component

### LINKED ISSUE #1688: [UI] Provider dropdown acting weird
**Reserved** for [outreachy](https://www.outreachy.org/) candidates. In order for this to be assigned to you, make sure to complete the first step of the application process outlined in the [project description](https://www.outreachy.org/december-2019-to-march-2020-internship-round/communities/mozilla/#build-a-component-that-can-display-a-json-schema).

---

**To Reproduce**
1. Go to http://localhost:5080/worker-manager/create
2. Find the Provider dropdown and select 'aws'
3. See error in the code editor

Screen Shot 2019-10-18 at 2 59 13 PM

Screen Shot 2019-10-18 at 2 59 16 PM

**Expected behavior**

The code editor doesn't show an error.

**Additional context**

For step 2, if you select anything that isn't 'aws', it seems to work.

### PR DESCRIPTION
**Pull request by Jo-IE**
Hi @helfi92 , I found the problem with this issue was that the `aws` provider does not have a `workerPool.Config` value (it returns an empty string). Please let me know if what I have done is sufficient to resolve this issue.

Closes #1688

### CODE REVIEW

**Comment by djmitche** [comment]
@imbstack @owlishDeveloper do we need to update the initial value for the `google` providerType too, now that it has changed?

File: ui/src/components/WMWorkerPoolEditor/index.jsx [hunk 1]
```diff
@@ -394,7 +394,10 @@ export default class WMWorkerPoolEditor extends Component {
               }
               secondary={
                 <CodeEditor
-                  value={JSON.stringify(workerPool.config, null, 2)}
+                  value={
+                    JSON.stringify(workerPool.config, null, 2) ||
+                    JSON.stringify({})
+                  }
```
Comment (helfi92) [inline ui/src/components/WMWorkerPoolEditor/index.jsx]: `JSON.stringify(workerPool.config || {}, null, 2)`

**Comment by helfi92** [comment]
@Jo-IE We would need to add a _silent_ changelog to make the tests pass? `yarn changelog --silent --issue 1688` from the root directory

**Comment by helfi92** [comment]
Just waiting to hear back from @imbstack @owlishDeveloper before merging.

**Comment by Jo-IE** [comment]
ok :)

**Comment by imbstack** [comment]
Would probably be nice to change but doesn't need to happen in this PR. I can make an issue for it. Would it be nice to have more good-first-bugs or is that one so uninteresting as to make it unsuitable.

**Comment by helfi92** [comment]
A good first issue SGTM.

**Comment by Jo-IE** [comment]
Hi @helfi92 do you mean that what is written in the Code Editor is supposed to change when the value in the Provisioner dropdown changes?

**Comment by helfi92** [comment]
> Hi @helfi92 do you mean that what is written in the Code Editor is supposed to change when the value in the Provisioner dropdown changes?

Yeah.

**Comment by Jo-IE** [comment]
ok I'll work on that right now and send an updated PR.

**Pull request merged**